### DISCUSSION TITLE
Lemur plugin for EJBCA

### DISCUSSION THREAD

**Discussion by Dan33l** [Keyfactor/ejbca-ce#86] [Q&A]
Hi,
Accordingly with Primekey doc, [Primekey developed a plugin for Lemur](https://doc.primekey.com/ejbca/tutorials-and-guides/using-ejbca-as-a-certificate-management-system-cms#UsingEJBCAasaCertificateManagementSystem(CMS)-Lemur) .
I did not found it. It does not look to be provided by [Lemur plugins](https://github.com/Netflix/lemur/tree/master/lemur/plugins)

Is this plugin opensourced or yet available ? 
I wish you a beautiful day.

**Comment by Dan33l** [comment]
@mike-agrenius-kushner @primetomas or any body else that have peace of information about the EJBCA plugin for Lemur ?

**Reply by primetomas** [reply]
It's correct, there was a Lemur plug-in developed, and in use by a customer. It never got publicly published though, due to resource constraints, and I think it's using some Enterprise features, that may have been the case.
We haven't seen a lot of requests for Lemur, but our integrations team could probably answer some questions if it turns out to be a fact?

So the specific answer is, no it is not opensourced (we were talking about it), but it may be available from Keyfactor.

Cheers,
Tomas

**Reply by Dan33l** [reply]
Thank for this answer, it allows me to go one step further in my research.
First off all, I must specify that my organization :
 * works or deploys or pays for service exclusively with free software.
 * must be out of American laws to be GDPR compliant (we are in France).
Knowing this, I am looking at how to provide a certificate delivery service to human users (not web servers) who are part of several hundred different organizations (Local governments). And so a PKI (here EJBCA) and a CMS (Lemur, i don't know any other opensource CMS) looks to be evaluate IMO.

Let my know how your talks evolve. I saw that Primekey have a presence in France.

**Reply by primetomas** [reply]
Indeed, I can put you in contact with our office in Paris. Message me if you want.
For open source options, did you look at https://github.com/3KeyCompany/CZERTAINLY?
